### PR TITLE
[16.0][IMP] webservice: allow to return response object on http calls

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
you may need to use information on the headers of the response

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]
Code review :+1:

**Review by simahawk — COMMENTED** [review]
Do we really need a field for this?
Can't we have a param to achieve the same?
Maybe you can explain a bit more about your use case?

**Comment by JordiMForgeFlow** [comment]
@simahawk you mean to pass it as a parameter in the kwargs of the `_request` method?

The use case is basically that I need to extract some information of the headers of the response, so I need the result of the `_request` to not be only the content

**Comment by simahawk** [comment]
> @simahawk you mean to pass it as a parameter in the kwargs of the `_request` method?

Yes, eg: `content_only=True` or `full_response=False`.

> The use case is basically that I need to extract some information of the headers of the response, so I need the result of the `_request` to not be only the content

But it means that you don't do this programmatically and you are calling `backend.call` directly which means you have full control over the call, right?

**Comment by JordiMForgeFlow** [comment]
@simahawk yes, I am calling backend.call directly, so I should be able to pass it as a parameter. I will propose the change.

**Review by HviorForgeFlow — APPROVED** [review]
Some reference about content_only kwarg in the README would be nice :)

File: webservice/components/request_adapter.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class BaseRestRequestsAdapter(Component):
     # TODO: url and url_params could come from work_ctx
     def _request(self, method, url=None, url_params=None, **kwargs):
         url = self._get_url(url=url, url_params=url_params)
+        content_only = kwargs.pop("content_only", True)
```
Comment (MiquelRForgeFlow) [inline webservice/components/request_adapter.py:27]: reference this in readme :)

**Comment by JordiMForgeFlow** [comment]
@simahawk could we get this one merged? :)

**Comment by JordiMForgeFlow** [comment]
@simahawk could we get this one merged?

**Review by etobella — APPROVED** [review]

**Pull request merged**